sources/packages/backend/apps/db-migrations/src/sql/Announcements/Create-announcements-table.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE sims.announcements (
+    id SERIAL PRIMARY KEY,
+    message_title VARCHAR(100) NOT NULL,
+    message VARCHAR(200) NOT NULL,
```

Comment:
@katrina-aot-git I have just talked with Carly and I suggested increasing this to 500.
The ticket is updated.
I also updated the ticket to settle on the 100 for the `message_title`.

Reply:
updated